### DISCUSSION TITLE
How to use mysql master-slave in prisma？

### DISCUSSION THREAD

**Discussion by cheaprYang** [prisma/prisma#8047] [Q&A] [answered]
I need to use master-slave mode in certain scenarios

**Comment by janpio** [comment, marked as answer]
Unfortunately not support yet: https://github.com/prisma/prisma/issues/172 You can of course create two `PrismaClient` instances with different connection strings - but I assume that is not what you want.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
